src/main/java/core/basesyntax/NumberSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class NumberSupplier {
+    public static int getRandomNumber() {
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/12864128/fb1abedd-ec5e-441f-af95-46403e261f3f)

Reply:
I need create object with field  int randomNumber and use this object in class Lottery?

Reply:
Yes, you should create objects of Number and Color Suppliers in Lottery class. And object of Lottery class in Main method